test/integration/test_bulk_updates.py

```diff
@@ -0,0 +1,116 @@
+import time
+from unittest import TestCase
+
+import requests
+from ingest.api.ingestapi import IngestApi
+from ingest.downloader.data_collector import DataCollector
+from ingest.downloader.downloader import XlsDownloader
+from ingest.importer.importer import XlsImporter
+from openpyxl.worksheet.worksheet import Worksheet
+
+from broker.submissions import ExportToSpreadsheetService
+
+# TODO test the following flow:
+# import new spreadsheet -> download spreadsheet -> update spreadsheet -> import updated spreadsheet
+
+
+class BulkUpdateTest(TestCase):
+    def setUp(self) -> None:
+        # TODO add configuration to get ingest url from env variables
+        # TODO currently it is hardcode for dev API
+        self.ingest_url = "https://api.ingest.dev.archive.data.humancellatlas.org"
+        self.ingest_api = IngestApi(self.ingest_url)
+        # self.ingest_api = IngestApi()
+
+        self.export_service = ExportToSpreadsheetService(self.ingest_api)
+
+        self.importer = XlsImporter(self.ingest_api)
+        self.submission_url = None
+        self.data_collector = DataCollector(self.ingest_api)
+        self.downloader = XlsDownloader()
+
+    def test_download_spreadsheet(self):
+        # download the test spreadsheet for the new submission
+
+        # create a submission, track the submission url
+        self.submission_url = 'http://localhost:8080/submissionEnvelopes/60f570c9a8a292649e814171'
+
+        # download a spreadsheet
+        entity_list = self.data_collector.collect_data_by_submission_uuid('98ec9cbb-e76c-4f8f-9b97-98fce7b151cb')
+        flattened_json = self.downloader.convert_json(entity_list)
+        workbook = self.downloader.create_workbook(flattened_json)
+        workbook.save('downloaded-spreadsheet-original.xlsx')
+
+        # assert spreadsheet contains data from submission
+
+    def test_import_modify_and_export_spreadsheet_flow(self):
+        # TODO: add JWT token for auth - without it the import won't work
+        # download a spreadsheet given a submission uuid /<submission_uuid>/spreadsheet
+        submission_uuid = 'e9e923d8-30b5-4184-87c5-fae40ce01ac9'
+        submission = self.ingest_api.get_submission_by_uuid(submission_uuid)
+        submission_id = submission['_links']['self']['href'].split('/')[-1]
+        path_to_spreadsheet, spreadsheet = self.__export_spreadsheet(submission_uuid)
+
+        # validate same cell values and then modify it then check it again that it has been modified
```

Comment:
These steps can be made into a function like  `__export_spreadsheet` and `__update_project_title`